mappings/net/minecraft/client/util/PlayerKeyPairManagerImpl.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_mdilqcsv net/minecraft/client/util/PlayerKeyPairManagerImpl
+	FIELD f_thgwzahx PROFILEKEYS Ljava/nio/file/Path;
```

Comment:
hm, I don't know the context, but I feel like there should be an underline:
```suggestion
	FIELD f_thgwzahx PROFILE_KEYS Ljava/nio/file/Path;
```

Reply:
I named it like this because the path string is "profilekeys" (all lowercase), but I suppose it doesn't really matter